models/intermediate/int_jira__issue_epic.sql

```diff
@@ -43,59 +18,18 @@ order_epic_links as (
                 partition by issue_id order by updated_at desc
                 ) as row_num
 
-    from epic_history
+    from epic_field_history
 ),
 
 last_epic_link as (
 
-    select issue_id, epic_issue_id 
+    select 
+      issue_id, 
+      epic_issue_id 
     
     from order_epic_links
     where row_num = 1
-),
-
-grab_epic_name as (
-
-    select 
-        last_epic_link.issue_id,
-        last_epic_link.epic_issue_id,
-
-        issue_parents.issue_name as epic_name,
-        issue_parents.issue_key as epic_issue_key
-        
-    from last_epic_link 
-    -- to grab each epic's issue attributes
-        join issue_parents on last_epic_link.epic_issue_id = issue_parents.issue_id
-),
-
-issue_epics as (
-
-    select 
-        issue_parents.issue_id,
-        coalesce(issue_parents.parent_issue_key, grab_epic_name.epic_issue_key) as parent_issue_key,
-
-        coalesce(issue_parents.parent_issue_id, grab_epic_name.epic_issue_id) as parent_issue_id,
-        coalesce(issue_parents.parent_issue_name, grab_epic_name.epic_name) as parent_issue_name,
-        issue_parents.is_parent_epic or grab_epic_name.issue_id is not null as is_parent_epic
-
-    from issue_parents
-
-    left join grab_epic_name 
-        on issue_parents.issue_id = grab_epic_name.issue_id
-
-),
-
-final as (
-
-    select
-        issue_id,
-        parent_issue_id as epic_issue_id,
-        parent_issue_name as epic_name,
-        parent_issue_key as epic_key
-
-    from issue_epics 
-
-    where is_parent_epic
 )
 
-select * from final
\ No newline at end of file
+select *
+from last_epic_link
```

Comment:
wondering if we should change this model name to reflect the fact that it only contains issue-epic relationships from classic projects (maybe `int_jira__issue_epic_link`)